src/pages/HomePage.vue

```diff
@@ -0,0 +1,241 @@
+<template>
+  <div class="page">
+    <div class="vertical-flex-container banner-background">
+      <NavBar />
+      <div class="banner-title">
+        <div>
+          <p style="margin-left: -5vw; font-size: 7vw">MCGILL</p>
+          <div class="title-text">
+            <p>CHINESE STUDENTS &</p>
+            <p>SCHOLARS</p>
+            <p>ASSOCIATION</p>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <RecentEvents />
+
+    <div class="about-us">
+      <div id="about-us-left">
+        <img
+          id="about-us-image"
+          src="https://res.cloudinary.com/mcgillcssa/image/upload/s--ztjHHMAh--/q_jpegmini/v1574799607/cssa_jpqr5f.jpg"
+        />
+        <div class="vertical-flex-container">
+          <div id="about-us-description">
```

Comment:
The about us text is not fully displayed on the smaller window

Reply:
Added a quick fix that changes the layout from row to column on smaller screens. I will modify it to find a better solution later after the media design team verifies the page.